lib/og/main.js

```diff
@@ -43,6 +43,24 @@ document.body.querySelector('#collect').addEventListener('click', function() {
   var graph = new ObjectGraph({
     maxDequeueSize: 1000,
     onDone: function() {
+      // Freeze and copy browser info from UI.
+      var browserNameE = body.querySelector('#browser-name');
+      var browserVersionE = body.querySelector('#browser-version');
+      var platformNameE = body.querySelector('#platform-name');
+      var platformVersionE = body.querySelector('#platform-version');
+      browserNameE.disabled = true;
+      browserVersionE.disabled = true;
+      platformNameE.disabled = true;
+      platformVersionE.disabled = true;
+      graph.environment.browser.name =
+          body.querySelector('#browser-name').value;
```

Comment:
Can you use `browserNameE.value` to remove any doubt that it's the same element?